crates/runtime-request-context/src/context.rs

```diff
@@ -171,6 +174,11 @@ impl RequestContext {
         dimensions
     }
 
+    #[must_use]
+    pub fn to_protocol_dimensions(&self) -> Vec<KeyValue> {
```

Comment:
This will allocate on every query, there are a limited number of possible values here, can we statically allocate them on boot and just return static references to the appropriate dimensions?